frontend/package.json

```diff
@@ -81,7 +79,7 @@
     "jspdf": "^2.3.0",
     "lint-staged": "^10.0.8",
     "lodash": "^4.17.21",
-    "mapbox-gl": "npm:maplibre-gl@1.15.2",
+    "maplibre-gl": "^2.0.0",
```

Comment:
Why are we not pointing to the latest versions directly? It seems that the latest version is 3.6.2?

Reply:
v2 seems to be fixing the build, maybe that is why they have it like that in their example: https://github.com/visgl/react-map-gl/tree/master/examples/get-started/maplibre.